packages/story-editor/src/components/canvas/utils/useFullbleedMediaAsBackground.js

```diff
@@ -39,19 +39,26 @@ function useFullbleedMediaAsBackground({ selectedElement }) {
   const { showSnackbar } = useSnackbar();
 
   const handleFullbleedMediaAsBackground = (target) => {
+    let isBackgroundMessageShown = localStorage.getItem('web_stories_is_background_message_shown');
```

Comment:
We are generally using `localStore` from the `design-system` package for managing the local storage. You could check for example about how we're using it by just searching for `localStore` in the `story-editor` package, one example is in this component: https://github.com/GoogleForCreators/web-stories-wp/blob/ad0e6193cdaea8bf3348fe8305c73f117140bcdb/packages/story-editor/src/components/colorPicker/confirmationDialog.js#L49

Since getting stuff from local storage is not very fast then it's also using state for it there.

Reply:
Done